src/main/java/org/packt/mygamelist/domain/GameRepository.java

```diff
@@ -1,8 +1,19 @@
 package org.packt.mygamelist.domain;
 
 import org.springframework.data.repository.CrudRepository;
-import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
 
-@RepositoryRestResource
+import java.util.List;
+import java.util.Optional;
+
+@Repository
 public interface GameRepository extends CrudRepository<Game, Long> {
+    Optional<Game> findByName(@Param("name") String name);
+
+    List<Game> findByIsGameAvail(@Param("isGameAvail") boolean isGameAvail);
+
+    List<Game> findByPrice(@Param("price") int price);
+
+    void deleteByName(@Param("name") String name);
```

Comment:
Is it working if you remove @Param annotations?

Reply:
It does